app/stylesheets/global.scss

```diff
@@ -0,0 +1,12 @@
+@import 'config/timing';
+@import 'config/type';
+
+html,
+body {
+  font-family: 'Open Sans', sans-serif;
+}
+
+a {
+  @include link-transition( 600ms, ease);
```

Comment:
Lets either
a) make `link-transition` have default values for timing (imported from `timing`) so that here we would just have `link-transiton()`, **or**
b) drop the `link-transition` mixin and just use variables with regular `transition`, like
`transition: colour $link-duration $link-easing;`.